src/bin/sage-env

```diff
@@ -277,6 +277,15 @@ export UNAME=`uname | sed 's/CYGWIN.*/CYGWIN/' `
 # Mac OS X-specific setup
 if [ "$UNAME" = "Darwin" ]; then
     export MACOSX_VERSION=`uname -r | awk -F. '{print $1}'`
+    # Try to identify command-line tools version.
+    # See https://apple.stackexchange.com/q/180957/351985.
+    XCLT_VERSION=`pkgutil --pkg-info=com.apple.pkg.CLTools_Executables | awk '/version: / { print $NF }' | cut -d. -f-1`
```

Comment:
I'm not sure if querying with `pkgutil` is the right approach. 
I think on a system several versions of CLT (and separately several versions of Xcode) can be installed.